### ISSUE TITLE
Add per network RPC endpoint registries

### ISSUE THREAD

**Issue by PierreBesson** [paritytech/polkadot_network_directory#49]
We currently have all rpc endpoints retrievable in a big JSON at https://paritytech.github.io/polkadot_network_directory/registry.json
It would be useful to have individual links for each network with: `/polkadot_network_directory/{network}/registry.json`. This could be integrated by third parties to have a ready to use list of RPC providers.

Ideally this format would also be standardized as it is used by [SubRPC](https://github.com/chevdor/subrpc)

**Comment by chevdor** [comment]
Yes, I was indeed consiedering this. I think this is out of scope for this repo but could be automated in `subrpc`.
Some people will want a json per network, some per provider, etc...

I am [currently working](https://github.com/chevdor/subrpc/tree/wk-230216-filters) on implementing filtering in subrpc. Using this, we would be able to feed ALL we have into subrpc, then use filtering to get only one network or one provider and re-export into a json file. The rest is mostly CI.

**Comment by chevdor** [comment]
Yes. I think the best approach is:
- we keep feeding data into the directory, either in this repo, or in subrpc. This is important because filtering is irrelevant if we don't have the data to filter in the first place
- the data should be "clean" with proper tags as much as possible. Tags/Labels can be global or per endpoint
- I get the filtering out on the `subrpc` side
- we can then add some CI in the subrpc repo to re-export filtered lists

@PierreBesson note that we do NOT need separate json files per network as soon as we have filtering available as dApps could provide their own filters and select only specific providers or properties (ie http only, or ws only, w/ or w/o ssl, etc....)
This is what [this branch](https://github.com/chevdor/subrpc/tree/wk-230216-filters) is about.
